tests/integration/onchain_ops.go

```diff
@@ -0,0 +1,165 @@
+package integration
```

Comment:
All onchain operations (deployments and contract calls) are now implemented in this file.

I've also made them independent of the `testify` suite instance (which we've used for the integration tests) so that it's easy to reuse the operations in other test files.